packages/react-querybuilder/src/types/importExport.ts

```diff
@@ -128,6 +128,11 @@ export interface FormatQueryOptions {
    * @default '~'
    */
   placeholderOperatorName?: string;
+  /**
+   * Character to use for quoting string values in the SQL format.
+   * @default '''
+   */
+  quoteChar?: `'` | `"`;
```

Comment:
To align with the existing option `quoteFieldNamesWith`, let's name this one `quoteValuesWith`.

Reply:
One more comment: type it as `string` to be as flexible as possible. If you really want to get fancy you can type it as `"'" | '"' | (string & {})`, which effectively types it as `string` but suggests `'` and `"` for autocomplete in IDEs like VS Code.